src/extension/app/store/app.js

```diff
@@ -1332,8 +1333,6 @@ export class AppStore {
           this.setupPlugins();
           this.fetchStatus();
           this.fireEvent(EXTERNAL_EVENTS.LOGGED_OUT, this.status.profile);
-          // refresh the page to reflect logged out state
-          this.reloadPage();
```

Comment:
I don't see why this should be required. We don't reload the page on login, either.